packages/camera/camera/ios/Classes/FLTCam_Test.h

```diff
@@ -5,6 +5,14 @@
 #import "FLTCam.h"
 #import "FLTSavePhotoDelegate.h"
 
+@interface FLTImageStreamHandler : NSObject <FlutterStreamHandler>
+// The queue on which `eventSink` property should be accessed
```

Comment:
Nit: These are declaration comments, so should follow declaration comment style (which is `///` for this file).